gen/installer/aws.py

```diff
@@ -54,6 +54,83 @@
         'id': 'ap-southeast-2'
     }]
 
+
+region_to_ami_map = [
```

Comment:
as long as we're here want to fix https://dcosjira.atlassian.net/browse/DCOS-12?